### DISCUSSION TITLE
Show release notes on empty/tips screen

### DISCUSSION THREAD

**Discussion by squatto** [spatie/ray#224] [Ideas]
When the app first loads or when you clear your screen, right now you're showing the tips screen, which includes a link to the release notes for the current version. I'd love to see the release notes right on that screen rather than having to load them up in a browser. Since it's for a specific version I'd think you could pull the release notes once and cache them locally so that you aren't hitting an endpoint every time the screen comes up.

Thanks!

**Comment by freekmurze** [comment]
Thanks for the suggestion

We'll consider improving this 👍
